contracts/solana/README.md

```diff
@@ -52,3 +88,14 @@ anchor test
 ```
 anchor build
 ```
+#### Initializing contract
+
+```
+ANCHOR_PROVIDER_URL=http://localhost:8899 ANCHOR_WALLET=~/.config/solana/id.json npx ts-node ./app/admin.ts initialize <asset_id> <value>
```

Comment:
I don't think asset_id and value are required here

Reply:
Oh whoops, removing.